iron-ajax.html

```diff
@@ -458,6 +478,14 @@
       var request = /** @type {!IronRequestElement} */ (document.createElement('iron-request'));
       var requestOptions = this.toRequestOptions();
 
+      if (this.lastRequest) {
```

Comment:
This should be moved below, before setting the new lastRequest (notice there is a return condition if the `iron-ajax-presend` event is prevented)